ts/src/heap-profiler.ts

```diff
@@ -84,17 +83,17 @@ export function profile(
  * started with different parameters, this throws an error.
  *
  * @param intervalBytes - average number of bytes between samples.
- * @param stackDepth - maximum stack depth for samples collected.
+ * @param stackDepth - maximum stack depth for samples collected. This is currently no-op.
+ *                     Default stack depth of 128 will be used. Kept to avoid making breaking change.
```

Comment:
I'd favor making the breaking change here (over keeping behavior which no longer works).